Description: Drop alloc since rustc-std-worspace-alloc-dev isn't packaged
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2023-05-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -74,7 +74,2 @@ harness = false
 
-[dependencies.alloc]
-version = "1.0.0"
-optional = true
-package = "rustc-std-workspace-alloc"
-
 [dependencies.bitflags]
@@ -176,3 +171,2 @@ rustc-dep-of-std = [
     "core",
-    "alloc",
     "compiler_builtins",
